src/Optimizely/OptimizelyConfig/OptimizelyConfigService.php

```diff
@@ -207,6 +311,81 @@ protected function getVariationsMap(Experiment $experiment)
         return $variationsMap;
     }
 
+    /**
+     * Generates string of audience conditions mapped to audience names.
+     *
+     * @param array experiment audience conditions .
+     *
+     * @return string of experiment audience conditions.
+     */
+    protected function getAudiences(array $audienceCond)
```

Comment:
please name variables completely. this should be `audienceConditions`. Also, please rename the method, it returns audiences with stringified conditions.